test/namer_moviexml_test.py

```diff
@@ -107,21 +105,21 @@ def test_writing_xml_metadata_genre_flag(self):
     <type>Actor</type>
     <name>Carmela Clutch</name>
     <role>Female</role>
-    <image>{fakeTPDB.get_url()}flLf1pecTlKcpJCki30l5iWXNdQ=/1000x1500/smart/filters:sharpen():upscale()/https%3A%2F%2Fcdn.metadataapi.net%2Fperformer%2F3e%2Fb6%2F0a%2F9a33557155b8a6f3e6c88bee8ed65a6%2Fposter%2F6a00376cbee2aa83c8c2514b89b332276df42c35</image>
```

Comment:
these urls will get called during tests -- right now I let them fail, this should stay to keep dev machines from inadvertently from calling the real tpdb instance during tests.

Reply:
We don't have this images in local, besides it's from json which we get from api, so probably then better rewrite test

Reply:
I was letting the images fail after hitting local, the test doesn't verify they get downloaded.

To be fair I had one setup to work, just to be able to verify it was hit in code coverage.   The test must have lost verification of the image download in a refactor:

https://github.com/ThePornDatabase/namer/blob/main/test/utils.py#L152

Not super worried about, just letting you know why I put these in.

Reply:
I see, you added replace, but it just doesn't work, will work on it